.gitignore

```diff
@@ -5,3 +5,5 @@ __pycache__
 build/
 case_utils.egg-info/
 dist/
+.idea/
+test_files/
```

Comment:
NACK on line 9.  It'd be better to send an isolated-effect PR for `.idea/`, but I'm fine leaving that as a request for future PRs, so we can keep line 8.  Line 9, though, creates a hidden interface.

If you have test data to definitely never include, I suggest you track those files, but in a parent directory above wherever you cloned this repository.

Meanwhile, the `--recursive` flag should get a demonstration under `tests/case_utils/case_validate`, but let's settle on the behaviors you're looking for first (in a thread on the other patch).

Reply:
Sounds good, I'll make that change.